databuilder/tables/beta/tpp.py

```diff
@@ -197,3 +198,69 @@ class ons_cis(EventFrame):
     visit_num = Series(int)
     is_opted_out_of_nhs_data_share = Series(bool)
     last_linkage_dt = Series(datetime.date)
+
+
+@table
+class isaric(EventFrame):
```

Comment:
If we know this is all going to change then I might be tempted to give the table itself a name like `isaric_raw` or something. That would free you up to create an `isaric` table with the proper column types later without breaking anything written against the "raw" table.

Reply:
Good idea; will do.